projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -146,8 +155,8 @@ function wpcom_register_default_launchpad_checklists() {
 		array(
 			'id'                   => 'setup_blog',
 			'title'                => __( 'Name your blog', 'jetpack-mu-wpcom' ),
-			'is_complete_callback' => '__return_false',
-			'is_disabled_callback' => '__return_false',
+			'is_complete_callback' => 'wpcom_is_setup_blog_completed',
+			'is_disabled_callback' => 'wpcom_is_task_option_completed',
```

Comment:
```suggestion
			'is_complete_callback' => 'wpcom_is_task_option_completed',
```
Should it be like this suggestion?